docs/reference/cli.md

```diff
@@ -819,28 +819,15 @@ Disables [telemetry](../telemetry.md) when set to any non-empty value.
 
 `QW_DISABLE_TELEMETRY=1 quickwit help`
 
-<!--
-    End of auto-generated CLI docs
--->
+### RUST_LOG
 
-## Environment Variables
+Configure quickwit log level.
 
-### QW_CLUSTER_ENDPOINT
```

Comment:
Why are the `QW_CLUSTER_ENDPOINT` and `QW_CONFIG` sections going away?

Reply:
for some reason they got duplicated. You can see it by loading the few lines before, or by checking the bottom of https://quickwit.io/docs/reference/cli